### PR TITLE
feat:input feature done

### PR DESCRIPTION
**Pull request by mariyasada**
adding input feature to CL. do it review.

### CODE REVIEW

File: stylesheets/input/input.css [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    top:10rem;
+    left:1rem;
+}
+.success-validation
+{
+    border: 2px solid var(--medium-green);
+}
+.input-textbox:focus{
+    border: 3px solid var(--dark-blue);
+}
```
Comment (JantuDeb) [inline stylesheets/input/input.css]:
```
.input-textbox:focus{
    border: 3px solid var(--dark-blue);
    outline:none;
}
```
This should work now,

**Review by JantuDeb — CHANGES_REQUESTED** [review]
I have commented on the required changes for the border to work on focus.

File: htmlFiles/input.html [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+                                            style="width: 700px; height: 300px; border:0; transform: scale(1); overflow:hidden;"
+                                            sandbox="allow-scripts allow-same-origin">
+                                    </iframe>
+                                </div>   
+                </div>
+  <!------------------------------------------------code of input ends- ---------------------------- -->
+         </section>
+    </div>
+<!-- ---------------------------------------------------------javascript functionality------------------------------- -->
+ <script>
```
Comment (Raghav888) [inline htmlFiles/input.html:191]: make another file of js and use it here via script tag

**Review by Raghav888 — APPROVED** [review]
Good to merge

**Comment by krishnakant01** [comment]
I think you've set cursor: pointer for the main content, I would suggest keeping a normal cursor and using pointer only while hovering over clickable items. Apart from that, the code looks good to merge!

File: htmlFiles/input.html (hunk 2 above)
Comment (mariyasada) [reply htmlFiles/input.html:191]: i will try to add this

**Pull request merged**